README.md

```diff
@@ -1,2 +1,87 @@
-# dev-team
-Proposal for the Dev Team
+# Student Robotics Dev Team Proposal
+
+This document aims to solve the important issue of development by outlining the formation of a number of smaller sub-groups within Student Robotics:
+- The Development Coordination Team.
+- A number of smaller _project groups_.
+
+## The Development Coordination team
+
+The Development Coordination team (a.k.a. the _Dev Team_) is a group of volunteers that are chosen annually in a similar fashion to the formation of the _Core Team_, although they may be responsible for projects beyond the annual timescale.
+
+The goals of the _Dev Team_ are to ensure the long-term longevity of Student Robotics by working on projects that enhance the experience for SR’s competitors and volunteers in line with its values, and enable SR’s long term growth. Additionally they will aim to fulfill the values of Student Robotics, in particular our goal of reflecting reality by incorporating the latest technology whenever feasible.
```

Comment:
> The dev team should focus on all values of SR equally

I disagree; the nature of separate teams is to have a specific focus. If all the teams and groups worked on all of the values equally, then the teams would be homogeneous and wouldn't actually do different things.

It makes sense for different teams to focus more on different values, so long as across the organisation as a whole, those values are all met. The core team is not well suited to focus on the value of maintaining up to date technology, as that largely requires projects which fall outside its timescale - it is for this very reason that the dev team was devised. It therefore seems to make sense for the dev team to make up the deficiencies we have seen in the core team, rather than simply focusing on all values equally.

Reply:
> > The goals of the _Dev Team_ are to ensure the long-term longevity of Student Robotics by working on projects

> This reads to me like the dev team will be doing the work themselves rather than orchestrating the project groups?

```suggestion
The goals of the _Dev Team_ are to ensure the long-term longevity of Student Robotics by organising projects that enhance the experience for SR’s competitors and volunteers in line with its values, and enable SR’s long term growth. Additionally they will aim to fulfill the values of Student Robotics, in particular our goal of reflecting reality by incorporating the latest technology whenever feasible.
```

Reply:
> reflecting reality by incorporating the latest technology whenever feasible

This sounds rather strict - does this mean we have to switch to a new web framework twice a month? I would suggest something like more recent/modern technologies.

Reply:
I agree.

```suggestion
The goals of the _Dev Team_ are to ensure the long-term longevity of Student Robotics by working on projects that enhance the experience for SR’s competitors and volunteers in line with its values, and enable SR’s long term growth. Additionally they will aim to fulfill the values of Student Robotics, in particular our goal of reflecting reality by incorporating the recent technology whenever feasible.
```

Reply:
> by incorporating the recent technology whenever feasible

This is no longer grammatically correct.